NineChronicles.Headless.AccessControlCenter/AccessControlService/MutableSqliteAccessControlService.cs

```diff
@@ -39,6 +40,11 @@ public void AllowAccess(Address address)
 
         public List<Address> ListBlockedAddresses(int offset, int limit)
         {
+            if (limit > 30)
```

Comment:
Is there a particular reason why the limit is set to 30?

Reply:
There's no reason for sqlite to have such a low limit...
Do you think a maximum of 100 is acceptable?